scripts/ci/check-for-version-info-cells.py

```diff
@@ -0,0 +1,46 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2025.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Check all notebooks have exactly one `version-info` cell.
+"""
+
+from pathlib import Path
+import nbformat
+import sys
+
+guides_path = Path("./docs/guides")
```

Comment:
We should look for all notebooks, such as tutorials. I recommend `Path("./docs").rglob("*.ipynb")`

Reply:
Are you sure? I think we should check with the tutorials team before we enforce this on them

Reply:
That's fair